back-end/api/api/urls.py

```diff
@@ -9,6 +9,7 @@
 router.register(r'authors', views.AuthorViewSet)
 router.register(r'posts', views.PostViewSet)
 router.register(r'followers', views.FollowersViewSet)
+router.register(r'likes', views.LikesPostViewSet)
 
```

Comment:
missed LikesCommentViewSet?

Reply:
No, only need to register it for one. They both present the same thing (likes). I added a comment about this.